.github/workflows/pr-check.yaml

```diff
@@ -135,10 +135,22 @@ jobs:
         working-directory: ./podman-desktop-extension-ai-lab
         run: pnpm install --no-frozen-lockfile
 
+      - name: Build Image
+        working-directory: ./podman-desktop-extension-ai-lab
+        id: build-image
+        run: |
+          pnpm build
+          podman build -t local_ai_lab_image ./
+          CONTAINER_ID=$(podman create localhost/local_ai_lab_image --entrypoint "")
+          podman export $CONTAINER_ID > /tmp/local_ai_lab_extension.tar
+          mkdir -p tests/playwright/tests/playwright/output/ai-lab-tests-pd/plugins
+          tar -xf /tmp/local_ai_lab_extension.tar -C tests/playwright/tests/playwright/output/ai-lab-tests-pd/plugins/
```

Comment:
can you do export and pipe to untar it rather than using an intermediate file

```
docker export $CONTAINER_ID | tar -x -C tests/playwright/tests/playwright/output/ai-lab-tests-pd/plugins/
```

also need to cleanup the remove the container

Reply:
fixed.